src/dotnet/commands/dotnet-run/RunCommandParser.cs

```diff
@@ -22,6 +22,7 @@ internal static class RunCommandParser
                     (
                         configuration: o.SingleArgumentOrDefault("--configuration"),
                         framework: o.SingleArgumentOrDefault("--framework"),
+                        runtime: o.SingleArgumentOrDefault("--runtime"),
```

Comment:
I'm missing where the following happens:

"Additionally, this PR adds the -r short form of the --runtime option so that it is consistent with the other commands."

Reply:
Yeah, I should have separated this out into its own commit.  I'm still happy to do so if we feel it's necessary.  One of those things that you run into while developing and you forget to stash, but it's related to the work you're doing so you elmoshrug.gif.

Reply:
No worries :)